src/dataframe/dataframe.jl

```diff
@@ -887,12 +887,11 @@ function Base.deleteat!(df::DataFrame, inds::AbstractVector{Bool})
     if length(inds) != size(df, 1)
         throw(BoundsError(df, (inds, :)))
     end
-    drop = _findall(inds)
     # workaround https://github.com/JuliaLang/julia/pull/41646
     if VERSION <= v"1.6.2" && drop isa UnitRange{<:Integer}
-        drop = collect(drop)
+        drop = collect(inds)
```

Comment:
This is no longer used.

Reply:
It is used, but cannot be tested. The point is that under Julia 1.6.2 we have:
```
julia> x = [1, 2]
2-element Array{Int64,1}:
 1
 2

julia> deleteat!(x, false:true) # error
ERROR: BoundsError: attempt to access 2-element Array{Int64,1} at index [0]

julia> deleteat!(x, collect(false:true)) # why we need this
1-element Array{Int64,1}:
 1
```

Since we still allow Julia 1.6.2 I thought to keep it. What do you think?

Reply:
I mean that `drop` isn't used anywhere now.

Reply:
ah - right. fixed